src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -4393,6 +4457,114 @@ partial class C
                 Diagnostic(ErrorCode.ERR_MemberNameSameAsType, "C").WithArguments("C").WithLocation(4, 18));
         }
 
+        [Fact]
```

Comment:
Sure. Thank you!

---
In reply to: [128396148](https://github.com/dotnet/roslyn/pull/20931#discussion_r128396148) [](ancestors = 128396148)